src/odemis/driver/test/smaract_test.py

```diff
@@ -68,14 +70,15 @@
                 'unit': 'rad',
             },
             'ry': {
-                'range': [-0.35, 0.35],
+                'range': [-0.45, 0.35],
```

Comment:
Is it different i.e. assymetrical rotation in ry?

Reply:
same remark for x

Reply:
I made it different so that it's easy to check which axis got which range in the test cases. There is no physical change in the system... and that doesn't matter much because it's mostly for the simulator. However, I realise that this test case could run on the real hardware, so I'm going to adjust the axes to be smaller than the actual hardware range.